app/services/app-loading.js

```diff
@@ -23,7 +23,11 @@ export default Service.extend({
   fetchTravisStatus: task(function* () {
     if (statusPageStatusUrl) {
       try {
-        const { status = {} } = yield $.get(statusPageStatusUrl) || {};
+        const { status = {} } = yield this.api.get(statusPageStatusUrl, {
```

Comment:
The same here about using `ajax` instead of `api`.

Reply:
Change made :)